example/lib/advanced/rpg_menu/bloc/rpg_menu_state.dart

```diff
@@ -0,0 +1,28 @@
+part of 'rpg_menu_cubit.dart';
+
+enum Category {
+  items,
+  magic,
+  weapon,
+  armor,
+  status,
+}
+
+enum Char {
+  dash,
+  ember,
+  hank,
+}
+
+class RpgMenuState extends Equatable {
+  const RpgMenuState({
+    this.category,
+    this.char,
+  });
+
+  final Category? category;
+  final Char? char;
+
+  @override
+  List<Object?> get props => [];
```

Comment:
```suggestion
  List<Object?> get props => [category, char,];
```

Reply:
Lol, good catch, you just saved me a couple of dozen minutes in the future trying to understand why my UI wasnt reacting to state changes